service/src/java/org/apache/hive/service/cli/session/SessionManager.java

```diff
@@ -224,7 +224,7 @@ private void createBackgroundOperationPool() {
     // Threads terminate when they are idle for more than the keepAliveTime
     // A bounded blocking queue is used to queue incoming operations, if #operations > poolSize
     String threadPoolName = "HiveServer2-Background-Pool";
-    final BlockingQueue queue = new LinkedBlockingQueue<Runnable>(poolQueueSize);
+    final BlockingQueue<Runnable> queue = new LinkedBlockingQueue<Runnable>(poolQueueSize);
```

Comment:
You can use <> here

Reply:
I'm going to commit these changes without this one change to save us from having to re-build :)